src/compile/data/bin.ts

```diff
@@ -126,7 +126,7 @@ export class BinNode extends DataFlowNode {
    * Creates a bin node from BinTransform.
    * The optional parameter should provide
    */
-  public static makeFromTransform(t: BinTransform, params: {model: Model} | {signal?: string, extentSignal?: string}) {
```

Comment:
@kanitw note that I just removed this and things still compile. Not sure why this was needed.

Reply:
someone forgot to remove it?

Reply:
Use blame to check?